src/main/java/com/fasterxml/jackson/annotation/JsonFormat.java

```diff
@@ -78,6 +80,15 @@
      */
     public final static String DEFAULT_TIMEZONE = "##default";
 
+    /**
+     * Value that indicates the default radix(numeric base) should be used when outputting
```

Comment:
Maybe rather explain this is marker/placeholder value for "use configured defaults" which typical mean "10"?